spur_bringup/launch/minimal.launch

```diff
@@ -18,8 +18,9 @@
   </include>
 
   <group unless="$(arg sim)" >
-    <node name="urg" pkg="urg_node" type="urg_node">
+    <node name="urg" pkg="hokuyo_node" type="hokuyo_node">
```

Comment:
although I think urg_node is less stable (and well developed) than hokuyo_node, for missing error check or warning comment, Okada-sensei said urg_node works fine, so better to keep using urg_node.